src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerIssueApplication.java

```diff
@@ -67,13 +62,13 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .pageLabel("Issue Divorce Application")
             .label("LabelNFDBanner-IssueApplication", SOLICITOR_NFD_PREVIEW_BANNER)
             .complex(CaseData::getApplication)
-                .complex(Application::getMarriageDetails)
-                    .optional(MarriageDetails::getDate)
-                    .optional(MarriageDetails::getApplicant1Name)
-                    .optional(MarriageDetails::getApplicant2Name)
-                    .mandatory(MarriageDetails::getPlaceOfMarriage)
-                    .done()
-                .done();
+            .complex(Application::getMarriageDetails)
+            .optional(MarriageDetails::getDate)
+            .optional(MarriageDetails::getApplicant1Name)
+            .optional(MarriageDetails::getApplicant2Name)
+            .mandatory(MarriageDetails::getPlaceOfMarriage)
+            .done()
+            .done();
```

Comment:
formatting ?

Reply:
Intellij keeps on doing that, very annoying